.build/build-resolver.xml

```diff
@@ -20,8 +20,8 @@
          xmlns:unless="ant:unless">
 
     <!-- details of what version of Resolver ANT Tasks to fetch -->
-    <property name="resolver-ant-tasks.version" value="1.3.0" />
-    <property name="resolver-ant-tasks.local" value="${local.repository}/org/apache/maven/resolver/maven-resolver-ant-tasks/${resolver-ant-tasks.version}/maven-resolver-ant-tasks-${resolver-ant-tasks.version}.jar"/>
+    <property name="resolver-ant-tasks.version" value="1.5.0" />
+    <property name="resolver-ant-tasks.local" value="${local.repository}/org/apache/maven/resolver/maven-resolver-ant-tasks/${resolver-ant-tasks.version}/maven-resolver-ant-tasks-${resolver-ant-tasks.version}-uber.jar"/>
```

Comment:
why we updating the resolver version in this PR ?

Reply:
without any particular reason, I can revert if you insist, but on the other hand, there are not much PRs which will be applied to all branches starting from 3.0. And this is actually quite painful, so if we are touching the build I thought that we can do such minor changes.

Reply:
this is discouraging, but ok :)